smartcar.go

```diff
@@ -102,7 +103,7 @@ func (c *client) IsTokenExpired(params *TokenExpiredParams) bool {
 
 // IsVINCompatible checks if a VIN is compatible for a list scopes.
 func (c *client) IsVINCompatible(ctx context.Context, params *VINCompatibleParams) (bool, error) {
-	url := buildCompatibilityURL(params.VIN, params.Scope)
+	url := buildCompatibilityURL(params.VIN, params.Scope, params.Country)
```

Comment:
we should default it to `US` if not passed to match the behavior in the other SDKs